src/sentry/static/sentry/app/components/platformPicker.tsx

```diff
@@ -59,7 +59,7 @@ class PlatformPicker extends React.Component<Props, State> {
 
     const filtered = platforms
       .filter(this.state.filter ? subsetMatch : categoryMatch)
-      .sort((a, b) => a.id.localeCompare(b.id));
+      .sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
```suggestion
      .sort((a, b) => a.id.localeCompare(b.id) || a.name.localeCompare(b.name));
```

Reply:
This way it will sort by id first and then sort alphabetically by name inside the same main platform.

Reply:
Probably won't help however, since the `id` is unique, so it will always use that to sort

Reply:
Ah, right. Never mind then.